experimental/packages/opentelemetry-sdk-metrics-base-wip/README.md

```diff
@@ -0,0 +1,38 @@
+# WIP OpenTelemetry Metrics SDK
+
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry metrics allow a user to collect data and export it to a metrics backend like [Prometheus](https://prometheus.io/).
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/sdk-metrics-base
```

Comment:
```suggestion
npm install --save @opentelemetry/sdk-metrics-base-wip
```

Or I suppose this will be renamed in #2598?

Reply:
The package is private and will not be released until it is ready. At that time it will be renamed to remove the -wip. The only reason i put that in the name is to protect against an accidental publish so it has a different name than the "real" package and we don't break anything